Missing dependency in @rollup/plugin-babel installation command

Hey there,

I went through the setup instructions for using babel with rollup and ran into an error. Running all installation commands and setting up the config file results in the following error:
```sh
[!] Error: Cannot find module '@babel/core'
Require stack:
- /root/dev/babel-rollup-example/node_modules/@rollup/plugin-babel/dist/index.js
- /root/dev/babel-rollup-example/rollup.config.js
- /root/dev/babel-rollup-example/node_modules/rollup/dist/shared/loadConfigFile.js
- /root/dev/babel-rollup-example/node_modules/rollup/dist/bin/rollup
```

Looking over at Rollup's docs, their set up instructions for babel includes `npm i -D @babel/core @babel/preset-env` [https://rollupjs.org/guide/en/#babel](https://rollupjs.org/guide/en/#babel)

Can submit a PR in a bit to fix this!

Thanks!

--- a/website/data/tools/rollup/install.md
+++ b/website/data/tools/rollup/install.md
@@ -1,3 +1,3 @@
 ```sh
-npm install --save-dev @rollup/plugin-babel
+npm install --save-dev @rollup/plugin-babel @babel/core
 ```

